// juniper.afi_tree is generated by proto_generator as a protobuf
// representation of a YANG schema.
//
// Input schema modules:
//  - afi-cap-action.yang
//  - afi-cap-entry-action.yang
//  - afi-cap-entry-match.yang
//  - afi-cap-entry.yang
//  - afi-cap-match.yang
//  - afi-cap.yang
//  - afi-common.yang
//  - afi-cos-classifier.yang
//  - afi-counter.yang
//  - afi-decap.yang
//  - afi-discard.yang
//  - afi-encap-entry.yang
//  - afi-encap.yang
//  - afi-filter.yang
//  - afi-fwd-sandbox.yang
//  - afi-indexed-list.yang
//  - afi-indirect.yang
//  - afi-list.yang
//  - afi-match.yang
//  - afi-oam.yang
//  - afi-policer.yang
//  - afi-port.yang
//  - afi-receive.yang
//  - afi-replicate.yang
//  - afi-sample.yang
//  - afi-sandbox.yang
//  - afi-selector.yang
//  - afi-switch.yang
//  - afi-table.yang
//  - afi-tree-encap-entry.yang
//  - afi-tree-encap.yang
//  - afi-tree-entry.yang
//  - afi-tree.yang
//  - afi-types.yang
//  - afi.yang
syntax = "proto3";

package juniper.afi_tree;

import "ywrapper/ywrapper.proto";
import "yext/yext.proto";
import "juniper/enums/enums.proto";

message AfiTree {
  ywrapper.StringValue default_next_node = 188585364;
  ywrapper.StringValue key_field = 32307618;
  repeated juniper.enums.AfiTreeAfiField key_fields = 168622709;
  ywrapper.StringValue name = 71110621;
  ywrapper.UintValue proto = 228442342;
  ywrapper.UintValue size = 287392455;
  juniper.enums.AfiTreeAfiTreeType type = 184483852;
}
